src/utils/Draft.js

```diff
@@ -281,6 +290,9 @@ async function finish() {
       const colors = botColors(null, null, draft.seats[i].pickorder, cards);
       draft.seats[i].name = `Bot ${i + 1}: ${colors.join(', ')}`;
       draft.seats[i].description = `This deck was drafted by a bot with color preference for ${colors.join('')}.`;
+    } else {
+      const colors = botColors(null, null, draft.seats[i].pickorder, cards);
+      draft.seats[i].name = `${draft.seats[i].name}: ${colors.join(', ')}`;
```

Comment:
No description?

Reply:
Description gets added by the user, we're just modifying it so the name includes our estimated colors of the deck like it does for the bot.